### DISCUSSION TITLE
en/blog/en/how-to-create-your-blog-for-free-by-hugo-ladder-in-30min/

### DISCUSSION THREAD

**Discussion by giscus** [guangzhengli/blog-comments#7] [Announcements]
# en/blog/en/how-to-create-your-blog-for-free-by-hugo-ladder-in-30min/

Ladder@This article is the second in a series on how to create you blog by hugo ladder for free, including using the hugo ladder configuration and integrate comments and analytics features, etc.

https://guangzhengli.com/en/blog/en/how-to-create-your-blog-for-free-by-hugo-ladder-in-30min/

**Comment by SaulJWu** [comment]
这个英文模式是怎么回事啊，自动机翻？牛逼

**Reply by guangzhengli** [reply]
那肯定还是需要手动写英文文章的[doge]

**Reply by SaulJWu** [reply]
不能楼中楼回复么，话说，两边空白是不是该有个路由跳转

**Reply by guangzhengli** [reply]
评论是基于 GitHub Discussions，目前算是楼中楼吧。

目前为了加强阅读体验，特意使用了两边留白的设计，把 Table of content 放到了文章开头，不过未来也可以设计成可选择式的放置。
